autosklearn/automl.py

```diff
@@ -1455,8 +1455,9 @@ def fit_ensemble(self, y, task=None, precision=32,
                      dataset_name=None, ensemble_nbest=None,
                      ensemble_size=None):
         # AutoSklearn does not handle sparse y for now
+        if ensemble_size == 0:
+            raise ValueError("ensemble_size must be greater than 0 for fit_ensemble") 
```

Comment:
Very small detail but match the check with the error message's statement :)
A little safer to have `> 0`